.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,8 +2,9 @@
 > Remember to delete this note before submitting your pull request.
 >
 > For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
-> 
-> For pull requests on Chinese localization, set Base Branch to release-1.15
+>
+> For pull requests on Chinese localization, set Base Branch to release-1.16
+> Feel free to ask questions in #kubernetes-docs-zh
```

Comment:
It'd be good to provide more info about the channel, for example `Feel free to ask questions in the #kubernetes-docs-zh Slack channel on [kubernetes.slack.com](https://kubernetes.slack.com)`